ui/app/components/send/currency-display.js

```diff
@@ -121,6 +127,19 @@ CurrencyDisplay.prototype.render = function () {
 
   const convertedValueToRender = this.getConvertedValueToRender(valueToRender)
 
+  function onlyRenderConversions() {
```

Comment:
I like that this was moved into a function. 👍

Reply:
Instead of declaring this function within `render()` can you add it to the prototype as is done with the other methods like `CurrencyDisplay.prototype.handleChange`?